internal/sink/sink.go

```diff
@@ -68,31 +67,32 @@ func NewSink(c Config, logger *zap.Logger, listener ancla.Register) Sink {
 			kafka := &Kafka{
 				id:         l.Registration.CanonicalName,
 				brokerAddr: k.BootstrapServers,
-				topic:      "test",
+				topic:      "quickstart-events",
+				logger:     logger,
 			}
 
+			config := sarama.NewConfig()
 			//TODO: this is basic set up for now - will need to add more options to config
 			//once we know what we are allowing users to send
-			kafka.config.Producer.Return.Successes = true			
-			kafka.config.Producer.RequiredAcks = sarama.WaitForAll
-			kafka.config.Producer.Retry.Max = c.DeliveryRetries //should we be using retryhint for this?
 
+			config.Producer.Return.Successes = true
+			config.Producer.RequiredAcks = sarama.WaitForAll
+			config.Producer.Retry.Max = c.DeliveryRetries //should we be using retryhint for this?
+			kafka.config = config
 			// Create a new Kafka producer
 			producer, err := sarama.NewSyncProducer(kafka.brokerAddr, config)
 			if err != nil {
 				kafka.logger.Error("Could not create Kafka producer", zap.Error(err))
 				return nil
 			}
-			defer producer.Close()
```

Comment:
this was moved to here: https://github.com/xmidt-org/caduceus/pull/541/files#diff-06f66c32b314a594bf3f237f16c59ea323846c805e5ae1e2d086e87e3b8edfc3R349